static/templates/navbar.hbs

```diff
@@ -1,73 +1,75 @@
 <div class="header">
-    <nav class="header-main rightside-userlist" id="top_navbar">
-        <div class="column-left">
-            <a class="brand no-style" href="#">
-                <img id="realm-logo" src="" alt="" class="nav-logo no-drag"/>
-            </a>
-        </div>
-        <div class="column-middle" id="navbar-middle">
-            <div class="column-middle-inner">
-                <div id="streamlist-toggle" {{#if embedded}}class="hide-streamlist-toggle-visibility"{{/if}} title="{{t 'Stream list' }} (q)">
-                    <a id="streamlist-toggle-button" role="button" tabindex="0"><i class="fa fa-reorder" aria-hidden="true"></i>
-                        <span id="streamlist-toggle-unreadcount">0</span>
-                    </a>
-                </div>
-                <div class="top-navbar-border top-navbar-container">
-                    <div id="message_view_header" class="notdisplayed">
+    <div class="header-wrapper">
```

Comment:
This change only adds this line and a closing tag below (can see if you view the diff hiding whitespace)

Reply:
Can you talk about why we need this `div`? I ask mainly because we've been accused in the past of having endless unnecessary layers of divs in our templates, so I'd like to understand why we can't just do this by applying classes to `header-main` or the `header` element around it.

(I'm also not sure about the naming scheme of `header` contains `header-wrapper` contains `header-main`, but that's probably secondary).

If we indeed need such an outer div, we should consider doing a prep commit to add it (with no visual changes) just to make the diff easier to read.

Reply:
Ah I inherited that change in this PR and hadn't thought about it critically. Great point, and yes it's not needed so I have removed it :)